app/importers/reading/mega_reading_processor.rb

```diff
@@ -107,7 +111,9 @@ def match_student(row)
     return [student.id, nil] if student.present?
 
     # fuzzy
-    last_first_name = row['Name']
+    last_first_name = row['student_last_names_first_names']
```

Comment:
The new template has the name, but it also has `student_local_id` and for now let's peg it to just that.  So that means we can use the key `student_local_id` rather than the other ones above, and that we can remove the fuzzy matching.

When I used the fuzzy matching previously trying this on the console, I would typically try processing, see all the errors, then go change the sheet to fix ambiguous names or typos, and re-run until it was correct.  I think that kind of workflow will be harder for educators to do themselves, or for us to make tools for.  So if we just use the lasid, then the code is simpler and the import rule is simpler: *to import data, you have to add the lasid to the sheet, here's how to look it up if you're not sure or here's a master sheet with all student names and lasids you can grab from* etc.

Reply:
addressed in https://github.com/studentinsights/studentinsights/pull/2544/files#diff-f01c15c5800dc5ef4b4a64b7ad07baadL100